### PR TITLE
TLS security

### PR DESCRIPTION
**Pull request by haiodo**
Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,2 @@
-HOST=localhost
 PORT=18081
 KMS_URL=ws://359.rocks:8888/kurento
```
Comment (sumboid) [inline .env]: BTW since file was changed, `359.rocks` also can be replaced to the `localhost`

File: distribution/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+version: "3.2"
+
+services:
+  meeting-server:
+    image: hardcoreeng/meeting-service
+    ports:
+      - 18081:18081
+      - 5000-5500:5000-5500
```
Comment (sumboid) [inline distribution/docker-compose.yaml]: Seems to be part of kms and there are also unnecessary `-` and below

**Review by sumboid — CHANGES_REQUESTED** [review]
It seems `package-lock.json` was generated by `npm`? Let's get rid of it in such case, we already have the `yarn.lock`

File: .env (hunk 1 above)
Comment (haiodo) [reply .env]: I've deployed with docker-compose only, so missed to rollback

File: distribution/docker-compose.yaml (hunk 2 above)
Comment (haiodo) [reply distribution/docker-compose.yaml]: Fixed

File: .env (hunk 1 above)
Comment (haiodo) [reply .env]: Fixed

**Review by sumboid — APPROVED** [review]

**Pull request merged**